### PR TITLE
Add DialogStudio

### PR DESCRIPTION
**Pull request by qbetterk**
add dialogstudio: summary: licenses, URLs, titles

### CODE REVIEW

File: data_summaries/DialogStudio.json [hunk 1]
```diff
@@ -0,0 +1,3826 @@
...
+    "Collection URL": "https://github.com/salesforce/DialogStudio",
+    "Languages": [
+      "'English'"
+    ],
+    "Task Categories": [
+      "'Context Generation'",
+      "'Dialogue Generation'",
+      "'Dialog Turn Prediction'",
+      "'Dialogue State Tracking'",
+      "'Dialogue Act Recognition'"
```
Comment (shayne-longpre) [inline data_summaries/DialogStudio.json:22]: Fix these strings as well

File: data_summaries/DialogStudio.json [hunk 2]
```diff
@@ -0,0 +1,3826 @@
...
+      ""
+    ],
+    "License": "MIT License",
+    "License URL": "https://github.com/asappresearch/abcd/blob/master/LICENSE",
+    "License Verified By": "Kun Qian",
+    "Licenses": {
+      "License": "MIT License",
+      "License URL": "https://github.com/asappresearch/abcd/blob/master/LICENSE"
+    },
+    "Dataset Filter IDs": ""
```
Comment (shayne-longpre) [inline data_summaries/DialogStudio.json:47]: This field can be used as a key to the dataset that you can pass directly into the huggingface dataset load function, or to filter for just examples from this dataset.

**Review by shayne-longpre — COMMENTED** [review]
Left some preliminary comments. Great work so far!

**Comment by qbetterk** [comment]
@shayne-longpre 
    fix summary file according to comments
    add download and prepare function

**Pull request merged**